src/app/_variables.scss

```diff
@@ -129,22 +129,15 @@ $emblem-height: 48px;
   }
 }
 
-/*
-Utility functions to allow for augmenting a hex color value with an alpha component.
-
-e.g.
-
-  --my-base-color-rgb: #{dim-hex-to-rgb-values(#ff7b00)};
-  background-color: dim-rgb-values-to-rgba(var(--my-base-color-rgb));
-  border-color: dim-rgb-values-to-rgba(var(--my-base-color-rgb), $alpha: 0.75);
-
-produces
-
-  --my-base-color-rgb: 255, 128, 0;
-  background-color: rgba(255, 128, 0, 1);
-  border-color: rgba(255, 128, 0, 0.75);
-
-*/
+// Utility functions to allow for augmenting a hex color value with an alpha component.
```

Comment:
I snuck this change in here, but as I was debugging I noticed that this big block comment was getting output in every file, which seemed unnecessary.